test/GithubApi.Issue.test.js

```diff
@@ -0,0 +1,73 @@
+const axios = require('axios');
+const { expect, assert } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+describe('Consume POST and PATCH Method', () => {
+  const user = 'MariaJose319';
+  const repo = 'BootcampQA_PetStoreChallenge';
+  const urlBase = 'https://api.github.com';
+  let userResponse;
+  let reposResponse;
+  let createIssueResponse;
+  let patchResponse;
+
+  before(async () => {
+    userResponse = await axios.get(`${urlBase}/user`, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+  });
+
+  it('verify that the user has at least a public repository', async () => {
+    expect(userResponse.status).to.equal(StatusCodes.OK);
+    expect(userResponse.data.public_repos).to.satisfy((num) => num > 0);
+  });
+
+  before(async () => {
+    reposResponse = await axios.get(userResponse.data.repos_url, {
+      headers: {
+        Authorization: `token ${process.env.ACCESS_TOKEN}`
+      }
+    });
+  });
+
+  it('get a list of repositories and verify that one exists', async () => {
+    expect(reposResponse.status).to.equal(StatusCodes.OK);
+    const reposList = reposResponse.data;
+    const repoInfo = reposList.find((element) => element.name === repo);
+    assert.exists(repoInfo);
+  });
+
+  before(async () => {
+    createIssueResponse = await axios.post(`${urlBase}/repos/${user}/${repo}/issues`, {
```

Comment:
```suggestion
    createIssueResponse = await axios.post(`${urlBase}/repos/${userResponse.data.login}/${repo}/issues`, {
```
Take proffit of hypermedia :)